extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-safeframe.js

```diff
@@ -218,6 +218,13 @@ describes.realWin('DoubleClick Fast Fetch - Safeframe', realWinConfig, env => {
     });
 
     it('should get current geometry when safeframe fills amp-ad', () => {
+      sandbox./*OK*/stub(safeframeHost.baseInstance_,
+          'getPageLayoutBox').returns({
+        top: 0,
```

Comment:
Could you update this to return a different value than iframeBox? As it is, the asserts won't fail if you reverted the changes in safeframe-host.js.

Same for other tests.

Reply:
Done